rasa/engine/recipes/default_recipe.py

```diff
@@ -374,9 +374,11 @@ def _extra_config_from_cli(
             old_number_epochs = component_config.get(
                 EPOCHS, component.get_default_config()[EPOCHS]
             )
-            epoch_fraction = float(cli_parameters["finetuning_epoch_fraction"])
-
-            config_from_cli[EPOCHS] = math.ceil(old_number_epochs * epoch_fraction)
+            epoch_fraction = cli_parameters["finetuning_epoch_fraction"]
+            if epoch_fraction is not None:
+                config_from_cli[EPOCHS] = math.ceil(
```

Comment:
... why is this even computed here? Doesn't whatever receives the fraction parameter and the number of epochs downstream compute this (again)?

Reply:
Agree with @ka-bu,  I think `config_from_cli["finetuning_epoch_fraction"] = epoch_fraction` should be enough here

Reply:
In this case all this block doesn't have any sense — `config_from_cli["finetuning_epoch_fraction"]` will be equal to `cli_parameters["finetuning_epoch_fraction"]` after the config from CLI is extracted (line 363)
so I guess the reason to do this calculation is to fill `config_from_cli[EPOCHS]`, isn't it?

Reply:
Updated the code, not sure if `config_from_cli[EPOCHS]` needs to be removed, can you confirm?

Reply:
Sure, makes sense